lib/install.ps1

```diff
@@ -355,7 +355,17 @@ function args($config, $dir) {
 function run($exe, $arg, $msg, $continue_exit_codes) {
     if($msg) { write-host $msg -nonewline }
     try {
-        $proc = start-process $exe -wait -ea stop -passthru -arg $arg
+        #Check if arguments were provided in the manifest, and pass them to the installer if present
+        if ($arg)
```

Comment:
I wonder if there's a way to make this more than concise in Powershell without `if/else`. If there's no way to make ps simply ignore a null argument, maybe we could prepare the arguments and use `@` splatting?

If you're not sure about the PS, do you have an example of an installer that doesn't take arguments and throws an error if you do `-arg $null`? I don't mind taking a look.